api/src/owner.rs

```diff
@@ -2137,10 +2137,10 @@ where
 	/// // The proof will likely be exported as JSON to be provided to another party
 	///
 	/// if let Ok(p) = result {
-	///		let valid = api_owner.verify_payment_proof(None, &p);
-	///		if let Ok(_) = valid {
-	///		  //...
-	///		}
+	///     let valid = api_owner.verify_payment_proof(None, &p);
+	///     if let Ok(_) = valid {
+	///       //...
+	///     }
```

Comment:
What are you running on it to tidy up doc comment formatting? Rustfmt doesn't get it

Reply:
Did it manually...